static/src/javascripts/projects/common/modules/commercial/permutive.spec.js

```diff
@@ -1,28 +1,47 @@
 // @flow
 import { local } from 'lib/storage';
-import { getPermutiveSegments } from './permutive';
+import { getPermutiveSegments, getPermutivePFPSegments, _ } from './permutive';
 
 jest.mock('lib/storage', () => ({
     local: {
         getRaw: jest.fn(),
     },
 }));
 
-describe('getPermutiveSegments', () => {
+afterEach(() => {
+    jest.clearAllMocks();
+});
+
+describe('getSegments', () => {
+    const DUMMY_KEY = `_dummyKey`;
     test('parses Permutive segments correctly', () => {
         local.getRaw.mockReturnValue(['[42,84,63]']);
-        expect(getPermutiveSegments()).toEqual(['42', '84', '63']);
+        expect(_.getSegments(DUMMY_KEY)).toEqual(['42', '84', '63']);
```

Comment:
I don't understand how this typechecks?! `getSegments` expects `key: SegmentsKey` but we're giving it `DUMMY_KEY`? The build passed so it's demonstrably "fine", but this is surprising to me.

Reply:
![](https://media.sketchfab.com/models/49efaa2c91094e4ab3c8184d2ec537a9/thumbnails/b0b040369cc541719272d827e328e643/7f49e2acb05a4beab23e908069e2ca22.jpeg)

Reply:
I don't think you need to change this, I'm just confused 😄

Reply:
@adamnfish I was also confused, I think`SegmentsKey` is basically an alias to type string.